.docker/test-lite.sh

```diff
@@ -1,13 +1,14 @@
 #!/bin/sh -e
 
-# Test PHP/Kimai
-# This does not work currently, as a real database is required
-# see https://github.com/kimai/kimai/issues/4503
-#/opt/kimai/bin/console kimai:version
-#if [ $? != 0 ]; then
-#  echo "PHP/Kimai not responding"
-#  exit 1
-#fi
+if [ -z "$DATABASE_URL" ]; then
+  export DATABASE_URL="mysql://kimai:kimai@127.0.0.1:3306/kimai?charset=utf8mb4&serverVersion=5.7.40"
+fi
+
+/opt/kimai/bin/console kimai:version
```

Comment:
`-z` tests the current process for an existing variable?

We should be able to do it like that as well:
```suggestion
DATABASE_URL=mysql://kimai:kimai@127.0.0.1:3306/kimai?charset=utf8mb4&serverVersion=5.7.40 /opt/kimai/bin/console kimai:version
```

Reply:
oh and thank you 😄

Reply:
-z checks if the variable is unset. The basic idea, from the script point of view, is:

if someone gives me a DATABASE_URL than that's ok, you should know better. Otherwise a DATABASE_URL is necessary so I'm filling a fake one.

Kimai already has a similar logic in the [startup ](https://github.com/kimai/kimai/blob/8eeb9c120a674532a75622b0ba322e4bd1f50a04/.docker/startup.sh) script (i.e. the entrypoint of normal execution):

```
  # Parse sql connection data
  if [ ! -z "$DATABASE_URL" ]; then
    DB_TYPE=$(awk -F '[/:@]' '{print $1}' <<< "$DATABASE_URL")
    DB_USER=$(awk -F '[/:@]' '{print $4}' <<< "$DATABASE_URL")
    DB_PASS=$(awk -F '[/:@]' '{print $5}' <<< "$DATABASE_URL")
    DB_HOST=$(awk -F '[/:@]' '{print $6}' <<< "$DATABASE_URL")
    DB_PORT=$(awk -F '[/:@]' '{print $7}' <<< "$DATABASE_URL")
    DB_BASE=$(awk -F '[/?]' '{print $4}' <<< "$DATABASE_URL")
  else
    DB_TYPE=${DB_TYPE:mysql}
    if [ "$DB_TYPE" == "mysql" ]; then
      export DATABASE_URL="${DB_TYPE}://${DB_USER:=kimai}:${DB_PASS:=kimai}@${DB_HOST:=sqldb}:${DB_PORT:=3306}/${DB_BASE:=kimai}"
    else
      echo "Unknown database type, cannot proceed. Only 'mysql' is supported, received: [$DB_TYPE]"
      exit 1
    fi
  fi
```

But if you prefer to override the value in any case, that's also possible. From a quick verification on the codebase the test-lite.sh appears to be called only by the CI/CD process.

Reply:
I verified the suggestion and it works. I would still keep the **if** verification (my impression is that I want to permit override from docker run command or whatever) so I would suggesto to modify it like this:

```suggestion
if [ -z "$DATABASE_URL" ]; then
  DATABASE_URL="mysql://kimai:kimai@127.0.0.1:3306/kimai?charset=utf8mb4&serverVersion=5.7.40"
fi
/opt/kimai/bin/console kimai:version
```
But last word is yours. Tell me how you prefer it and I will update it accordingly. Thank you.

Reply:
Thanks for your detailed testing and response. I like that!
Sounds as if your initial suggestion was already the best way to move forward. 
Not sure whether we want to use that script in another fashion, but its good to have the option to do so.

Thanks for all your input here. Very much needed, especially in the docker context, where I lack knowledge!

Reply:
Thank you too. I feel that we still got a better answer than my initial suggestion because removing the export command makes more sense for this script.
I will update the PR with an additional commit later today.

Reply:
Right?! In my experience discussions in PRs usually lead to better solutions and/or knowledge transfer. Both results are good 😄

Reply:
Oh, and I will wait for your change before merging ...